Automattic/vip
# Updating command option descriptions to follow the VIP-CLI style guide

Updated the voice, tense, and format for descriptions of several command options to follow the VIP-CLI style guide.

## Pull request checklist

- [ n/a ] Update [SETUP.md](https://github.com/Automattic/vip-cli/blob/trunk/docs/SETUP.md#list-of-environmental-variables) with any new environmental variables.
- [ x ] Follow the [pull request checklist](https://github.com/Automattic/vip-cli/blob/trunk/docs/RELEASING.md#new-pull-requests)
- [ x ] Add/update [automated tests](https://github.com/Automattic/vip-cli/blob/trunk/docs/TESTING.md#automated-testing) as needed.

## New release checklist


## Steps to Test

Observe the results of the `--help` display for impacted commands, such as:

- `vip dev-env logs -h`
- `vip dev-env update -h`
